providers/openstack/alpha/1-28/cluster-class/templates/cluster-class.yaml

```diff
@@ -167,6 +167,22 @@ spec:
          items:
            type: string
   patches:
+    - name: k8s_version
+      description: "Sets the openstack node image for workers and the controlplane to the cluster-api image with the version mentioned in spec.topology.version."
+      definitions:
+        - selector:
+            apiVersion: infrastructure.cluster.x-k8s.io/v1alpha7
+            kind: OpenStackMachineTemplate
+            matchResources:
+              controlPlane: true
+              machineDeploymentClass:
+                names:
+                  - {{ .Values.prefix }}-{{ .Values.testcluster_name }}
```

Comment:
Please rebase, in the new version this should be `{{ .Release.Name }}-{{ .Chart.Version }}`, also modify 1-29 clusterstacks if possible.